### PR TITLE
Implement gRPC submap query.

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer_grpc/handlers/get_submap_handler.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    : public framework::RpcHandler<proto::GetSubmapRequest,
+                                   proto::GetSubmapResponse> {
+ public:
+  void OnRequest(const proto::GetSubmapRequest &request) override {
+    auto response =
+        cartographer::common::make_unique<proto::GetSubmapResponse>();
+    response->set_error_msg(
+        GetContext<MapBuilderServer::MapBuilderContext>()
+            ->map_builder()
+            .SubmapToProto(cartographer::mapping::SubmapId{},
```
Comment (gaschler) [inline cartographer_grpc/handlers/get_submap_handler.h]: use the id from the request

Comment (cschuet) [reply cartographer_grpc/handlers/get_submap_handler.h]: Woopsie!

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**